composer.json

```diff
@@ -20,7 +20,7 @@
   },
   "require-release": {
```

Comment:
indent with 4 spaces, so the build changes/diff is simpler

also if atk4/ui dev-develop is wanted in require-dev - https://github.com/atk4/outbox/commit/cb95db683a2a9496d38dd20c91dea5c73258bf3c#diff-d2ab9925cad7eac58e0ff4cc0d251a937ecf49e4b6bf57f8b95aab76648a9d34R22 - there should be require-dev-release (and ideally test with "release/0.1" branch with GH CI)

Reply:
My fault, probably when I do the composer normalize something switch to 2 spaces.

what do you think if i do this?
```php
"require": {
        "php": ">=7.4 <8.2",
        "atk4/data": "dev-develop",
        "atk4/ui": "dev-develop",
        "phpmailer/phpmailer": "^v6.5.3"
    },
    "require-release": {
        "php": ">=7.4 <8.2",
        "atk4/data": "~3.2.0",
        "atk4/ui": "~3.2.0",
        "phpmailer/phpmailer": "^v6.5.3"
    },
```
Obviously, after that, I remove Atk4/Ui from require-dev, just to avoid spoofing in a library that follows the development of the two major packages of Atk4?

Reply:
I am fine with requiring atk4/ui

Reply:
you can also remove some dev deps required for Behat only which is not used